web/modules/custom/portland/modules/portland_groups/portland_groups.module

```diff
@@ -53,6 +53,39 @@ function _portland_groups_inline_entity_populate_field_display_groups(&$form, &$
   }
 }
 
+/**
+ * Implements hook_media_insert
+ */
+function portland_groups_media_insert($entity) {
+  _portland_groups_assign_media_to_group($entity);
+}
+
+/**
+ * Manually assigns a media entity to a group as group content.
+ * The group is identified by the first delta in the media's field_display_groups.
+ * This should only happen in the context of an inline entity form.
+ */
+function _portland_groups_assign_media_to_group($entity) {
+
+  // only do this for inline entity brower
+  $current_path = \Drupal::service('path.current')->getPath();
```

Comment:
How expensive will it be to call a function like this on every page? 🤔

Reply:
I really have no idea. It seems to be a pretty commonly used function, so I assumed it was negligible. It doesn't get called on every page, just whenever a media item is created. I can look into getting the current path/url using php server var.

Reply:
If you're concerned about the performance of that call, I can replace it with $_SERVER['REQUEST_URI'] without much hassle. Though I would expect that path/url operations are pretty lightweight and don't use any db calls or heavy string manipulation.

Reply:
No need to rewrite. I was just musing.